Fuzzing/common/MachOParse.mm

```diff
@@ -29,6 +29,8 @@ int get_num_fds() {
     // Mach-O Parsing
     [fi architectures];
     [fi isMissingPageZero];
+
```

Comment:
Why the empty line?

Reply:
I was originally distinguishing this from the pure Mach-O parsing but since the majority of info plist loading _is_ more binary parsing I'll remove to group it with the others.